clients/unshaded/src/main/java/tachyon/client/file/FileSystemMasterClient.java

```diff
@@ -364,6 +364,7 @@ public Boolean call() throws TachyonTException, TException {
    * Schedules the async persistence of the given file.
    *
    * @param fileId the file id
+   * @return RPC result of the scheduling
```

Comment:
I think the `Void` in the method signature here is supposed to be lowercase, no callers use the return value.